Mathlib/RingTheory/MatrixAlgebra.lean

```diff
@@ -146,5 +154,59 @@ theorem matrixEquivTensor_apply_stdBasisMatrix (i j : n) (x : A) :
 
 @[simp]
 theorem matrixEquivTensor_apply_symm (a : A) (M : Matrix n n R) :
-    (matrixEquivTensor R A n).symm (a ⊗ₜ M) = M.map fun x => a * algebraMap R A x :=
+    (matrixEquivTensor R A n).symm (a ⊗ₜ M) = a • M.map (algebraMap R A) :=
   rfl
+
+namespace Matrix
+open scoped Kronecker
+
+variable (m) (B) [Fintype m] [DecidableEq m]
+
+/--
+`Matrix.kroneckerTMul` as an algebra equivalence, when the two arguments are tensored.
+
+TODO:
+* Generalize this to rectangular matrices
+* Make `Matrix.kroneckerTMulAlgEquiv_tmul` true by `rfl`
+-/
+def kroneckerTMulAlgEquiv :
+    Matrix m m A ⊗[R] Matrix n n B ≃ₐ[R] Matrix (m × n) (m × n) (A ⊗[R] B) :=
```

Comment:
Do we care about the
```lean
    Matrix m m R ⊗[R] Matrix n n R ≃ₐ[R] Matrix (m × n) (m × n) R :=
```
special case, or is that easy enough to get by composition that we can skip it?

Reply:
I don't care personally